File: Assets/JavaScript/Game.js
#pragma strict

public class Game{
	public var nome: String;
	public var categoria: String;
	public var lancamento: String;
	
	public function Game(nome: String, categoria: String){
		this.nome = nome;
		this.categoria = categoria;
		this.lancamento = new Date().Now.ToString();
	}
}

File: Assets/C#/Game.cs
using UnityEngine;
using System.Collections;
using System;

public class Game {
	public string nome;
	public string categoria;
	public string lancamento;

	public Game(string nome, string categoria){
		this.nome = nome;
		this.categoria = categoria;
		this.lancamento = DateTime.Now.ToString();
	}
}
